Resources/Prototypes/Roles/Jobs/Cargo/salvage_specialist.yml

```diff
@@ -0,0 +1,24 @@
+- type: job
+  id: SalvageSpecialist
+  name: "salvage spationaut"
+  icon: "ShaftMiner"
+  startingGear: SalvageSpecialistGear
+  departments:
+  - Cargo
+  supervisors: "the quartermaster and head of personnel"
+  access:
+  - Cargo
+  - Salvage
+  - Maintenance
+  - External
+
+- type: startingGear
+  id: SalvageSpecialistGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitSalvageSpecialist
+    back: ClothingBackpackFilled
+    shoes: ClothingShoesBootsSalvage
+    id: SalvagePDA
+    ears: ClothingHeadsetCargo
+  satchel: ClothingBackpackSatchelFilled
+  duffelbag: ClothingBackpackDuffelFilled
```

Comment:
Could add fingerless gloves here on the basis that gets merged first. Would cause tests to fail until then so up to you.

Reply:
Black gloves would probably suit salvagers more, which I can add.